app/src/main/java/com/mapbox/maps/testapp/examples/MultiDisplayActivity.kt

```diff
@@ -0,0 +1,92 @@
+package com.mapbox.maps.testapp.examples
+
+import android.app.ActivityOptions
+import android.content.Context
+import android.content.Intent
+import android.hardware.display.DisplayManager
+import android.os.Build
+import android.os.Bundle
+import android.widget.Toast
+import androidx.appcompat.app.AppCompatActivity
+import com.mapbox.geojson.Point
+import com.mapbox.maps.CameraOptions
+import com.mapbox.maps.Style
+import com.mapbox.maps.plugin.animation.MapAnimationOptions.Companion.mapAnimationOptions
+import com.mapbox.maps.plugin.animation.flyTo
+import com.mapbox.maps.testapp.R
+import kotlinx.android.synthetic.main.activity_multi_display.*
+import kotlinx.android.synthetic.main.activity_simple_map.mapView
+
+/**
+ * Example of displaying a map on the second screen.
+ *
+ * To use this example, you need to attach a second display to your device, or
+ * have the simulated secondary display enabled in the developer settings.
+ */
+class MultiDisplayActivity : AppCompatActivity() {
+
+  override fun onCreate(savedInstanceState: Bundle?) {
+    super.onCreate(savedInstanceState)
+    setContentView(R.layout.activity_multi_display)
+    mapView.getMapboxMap().loadStyleUri(Style.DARK)
+    button.setOnClickListener {
+      displayMapInSecondaryScreen()
+    }
+    moveCameraButton.setOnClickListener {
+      moveCamera()
+    }
+  }
+
+  private fun moveCamera() {
+    val cameraOption = CameraOptions.Builder()
+      .center(Point.fromLngLat(24.9384, 60.1699))
+      .zoom(14.0)
+      .build()
+
+    mapView.getMapboxMap().flyTo(
+      cameraOption,
+      mapAnimationOptions {
+        duration = 1000
```

Comment:
I've checked that example - guess for better UX we could set a bit higher value here.
Also could move to const.

Reply:
Used 3000 instead.